build_protos.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2017 Google Inc.
```

Comment:
this essentially deprecates the 'makefile.py' which is currently in the root directory as well. Let's fix it as a follow-up issue.